src/utilities/Label.tsx

```diff
@@ -0,0 +1,24 @@
+import { LabelHTMLAttributes, HTMLAttributes } from "react";
+import { cn } from "../lib/utils/cn";
+
+type Props = LabelHTMLAttributes<HTMLLabelElement> &
+  HTMLAttributes<HTMLParagraphElement>
+
+
+export const Label = ({ htmlFor, children, className, ...res }: Props) => {
+    if (!htmlFor) {
+        return (
+          <p {...res} className={cn('inline-block text-md', className)}>
+            {children}
+          </p>
+        );
+      }
+    
+      return (
+        <label {...res} className={cn('inline-block text-md', className)}>
+          {children}
+        </label>
+      );
+};
+
+Label.displayName = 'Label';
```

Comment:
displayName is added because of the forwardRef. If we use forwardRef (which we need to use - explanation in the other comment), component is called "forwardRef" in react devtools. So it's better to set displayName explicitly for easier debugging.

Reply:
Oh, I've never seen this approach in my commercial apps :D Thanks for explanation